app/Http/Controllers/Band/BandController.php

```diff
@@ -15,4 +22,42 @@ public function showResetForm(Request $request, $token = null) {
 			['token' => $token, 'email' => $request->email]
 		);
 	}
+
+	public function show(Band $band) {
+		return view('band.band', compact('band'));
+	}
+
+	public function update(UpdateBandRequest $request, Band $band){
+		$request->commit();
+		return back()->with('toast', [
+			'type' => 'success',
+			'title' => '',
+			'message' => __('vue.updateSuccess', [], $request->input('language'))
+		]);
+	}
+
```

Comment:
is this necessary for  update band and update payment method to be different requests?

Reply:
mmm cuz its on a different tab

Reply:
I feel that it should be the same tab, with the extra tab appearing for members only if they are paid differently.

Reply:
ok, say it before :P